core/modules/views/views.install

```diff
@@ -192,6 +192,29 @@ function views_update_1007() {
   config_clear('views.settings', 'no_javascript');
 }
 
+/**
+ * Set all grid style Views to use the backwards-compatible unresponsive option
+ * if not already set.
+ */
+function views_update_1008() {
+  $view_names = config_get_names_with_prefix('views.view.');
+  foreach ($view_names as $view_name) {
+    $config = config($view_name);
+    $displays = $config->get('display');
+    foreach ($displays as $display => $display_details) {
+      if (!isset($display_details['display_options']['style_plugin']) || $display_details['display_options']['style_plugin'] != 'grid') {
+        continue;
+      }
+      if (!empty($display_details['display_options']['style_options']['unresponsive'])) {
+        continue;
+      }
+      $config->set('display.' . $display . '.display_options.style_options.unresponsive', 1);
+    }
+    $config->save();
+  }
+  cache_clear_all();
```

Comment:
This should be unnecessary. All caches are automatically cleared upon running update.php.

```suggestion
```